sopel/modules/reddit.py

```diff
@@ -195,6 +195,22 @@ def comment_info(bot, trigger, match):
 
 def subreddit_info(bot, trigger, match, commanded=False):
     """Shows information about the given subreddit"""
+    if match.lower() in ['all', 'popular']:
```

Comment:
Way too many `match.lower()` in here. You can work with an intermediate variable for this one I believe.

Reply:
assuming #1764 gets merged before this, I can rebase over that when the time comes.